framework/db/Command.php

```diff
@@ -457,9 +481,16 @@ public function insert($table, $columns)
      */
     public function batchInsert($table, $columns, $rows)
     {
+        $table = $this->db->quoteSql($table);
+        $columns = array_map(function ($column) {
+            return $this->db->quoteSql($column);
+        }, $columns);
+
         $sql = $this->db->getQueryBuilder()->batchInsert($table, $columns, $rows);
 
-        return $this->setSql($sql);
+        $this->setRawSql($sql);
```

Comment:
The intermediate variable `$sql` does little work. But its value could be a very long string. Unless the PHP compiler does The Right Thing then it's fine. Otherwise it `$sql` could be eliminated.

Reply:
I'm pretty sure it makes no difference so whatever style you prefer.